amazon-ses-connector-test/src/test/resources/docker-compose-test.yaml

```diff
@@ -0,0 +1,111 @@
+version: '3.9'
+
+x-sso-variables: &sso-variables
+  SIG_SECRET_KEY: JWTSecretKeyDontUseInProduction!
+  EXTERNAL_SIG_SECRET_KEY: JWTSecretKeyDontUseInProduction!
+  EXTERNAL_ENC_SECRET_KEY: JWTEncKeyDontUseInProduction666!
+  sso.jwt.internal.signature.key.type: ENV
+  sso.jwt.internal.signature.key.value: SIG_SECRET_KEY
+
+x-eureka-variables: &eureka-variables
+  eureka.client.service-url.defaultZone: http://registry:8080/eureka
+
+x-actuator-variables: &actuator-variables
+  management.endpoints.web.exposure.include: "*"
+  management.endpoints.enabled-by-default: "true"
+
+x-activemq-variables: &activemq-variables
+  spring.activemq.broker-url: tcp://message-broker:61616
+  spring.activemq.user: admin
+  spring.activemq.password: admin
+
+services:
+
+  registry:
+#    Use this one M1 ('custom-m1-b1' is a multi-arch image)
+#    image: "${REGISTRY_IMAGE:-harbor.backbase.eu/internal/registry:custom-m1-b1}"
+    image: "${REGISTRY_IMAGE:-repo.backbase.com/docker-releases/registry:2022.08.9-LTS}"
+    ports:
+      - "8761:8080"
+    environment:
+      JAVA_OPTS: -Xmx256m -Xms128m
+      <<: *sso-variables
+
+  message-broker:
+#    Use this one for M1
+#    image: "${MESSAGE_BROKER_IMAGE:-harbor.backbase.eu/internal/activemq:5.16.3-m1}"
+    image: "${MESSAGE_BROKER_IMAGE:-rmohr/activemq:latest}"
+    hostname: message-broker
+    ports:
+      - "61616:61616"
+      - "8161:8161"
+
+  token-converter:
```

Comment:
Same here, is the token converter really needed for the test? This will unnecessary spin up a non mandatory container making things slower.